components/ui/tags/CategoryTag.tsx

```diff
@@ -1,11 +1,12 @@
 interface TagProps {
   categoryName: string;
   categoryId: number;
+  className?: string;
 }
 
 // 카테고리 이름을 고유한 색으로 보여주는 태그 컴포넌트입니다.
 // 카테고리 id는 색상을 고유하게 만들기 위해서 사용합니다.
-function CategoryTag({ categoryName, categoryId = 1 }: TagProps) {
+function CategoryTag({ categoryName, categoryId = 1, className = '' }: TagProps) {
```

Comment:
```suggestion
function CategoryTag({ categoryName, categoryId = 1, className }: TagProps) {
```
전달된 값이 없을 때 undefined 그대로 두어도 괜찮을 것 같아요!